api/v1beta1/verticadb_types.go

```diff
@@ -39,12 +39,12 @@ const VerticaDBAPIVersion = "vertica.com/v1beta1"
 type VerticaDBSpec struct {
 	// +kubebuilder:validation:Optional
 	// +kubebuilder:default:=IfNotPresent
-	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors="urn:alm:descriptor:com.tectonic.ui:imagePullPolicy"
 	// This dictates the image pull policy to use
 	ImagePullPolicy corev1.PullPolicy `json:"imagePullPolicy,omitempty"`
 
 	// +kubebuilder:validation:Optional
-	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:io.kubernetes:Secret"}
```

Comment:
This takes a list of secrets.  Is it able to handle that?

Reply:
Yeah, it lets you specify a list of secrets but unlike other secret fields where it allows you to choose from a pool of existing secrets, here you have to manually write each secret name. So I think this x-descriptor does not apply here well because it is an array but I will see if there is a work around or if we have to let the user write them manually